config/storage.yml

```diff
@@ -6,6 +6,14 @@ local:
   service: Disk
   root: <%= Rails.root.join("storage") %>
 
+
+amazon:
+  service: S3
+  access_key_id: "AKIAJWD33W5RLW4OTKNA"
+  secret_access_key: "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"
```

Comment:
We must use the secrets

Reply:
@krlklim never ever commit sensitive info again, we've reset the credentials above.
Contact Nick for the new ones.